sdk/servicebus/service-bus/samples-dev/receiveMessagesLoop.ts

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT Licence.
+
+/**
+ * This sample demonstrates how the receiveMessages() function can be used to receive Service Bus
+ * messages in a loop.
+ *
+ * Setup: Please run "sendMessages.ts" sample before running this to populate the queue/topic
+ *
+ * @summary Demonstrates how to receive Service Bus messages in a loop
+ * @azsdk-weight 40
+ * @azsdk-ignore true
```

Comment:
`azsdk-ignore` causes this sample to be completely omitted from the output, are you sure that's what you want to do?